### DISCUSSION TITLE
MPGS - Mastercard Payment Gateway Services

### DISCUSSION THREAD

**Discussion by mlodhia10** [juspay/hyperswitch#5384] [Ideas / Feature Requests]
I am requesting the development of a new connector for the Mastercard Payment Gateway Service (MPGS) within the HyperSwitch platform. MPGS is a widely used payment gateway by most banks and financial institutions and integrating it as a connector will greatly benefit as banks are able to provide wholesale rates.

**Comment by KhurramChaudhry1** [comment]
Same i need for CS Cart, anyone can help please.

**Comment by egbordzor** [comment]
I need same for online (non-terminal) payments
